.travis.yml

```diff
@@ -1,13 +1,12 @@
 sudo: required
 dist: trusty
-language: cpp
+language: python
 
 python:
   - "2.7"
   - "3.4"
 
 before_install:
-  - export PATH=$(echo $PATH | tr ':' "\n" | sed '/\/opt\/python/d' | tr "\n" ":" | sed "s|::|:|g")
```

Comment:
Was this just unused?

Reply:
Frankly, I just didn't understand why we need this, but it seems to work without this line. I'm open to leaving this line as it is though.

Reply:
Ok, just checking.  We should remove it, it looks like it's trying to do some weird path resolution to an `/opt/python` binary which we don't have.